linelist_investigations/report_sources/active_health_areas_2019-07-12.Rmd

```diff
@@ -1,364 +1,385 @@
-
-```{r current active areas past 7 days}
-
-current_active_7_days <- x %>%
-  group_by(aire_de_sante) %>%
-  summarise(n_cases_7_days = sum(date_report > (database_date - 7))) %>%
-  filter(n_cases_7_days > 0) %>% 
-  left_join(aire_de_sante_info)
-
-
-ggplot(current_active_7_days, aes(x = aire_de_sante, y = n_cases_7_days)) +
-  geom_col(aes(fill = sous_coordination)) +
-  facet_grid(. ~ sous_coordination,
-             scales = "free_x",
-             space = "free") +
-  custom_horiz_facet +
-  large_txt +
-  theme(legend.position = "bottom",
-        axis.text.x = element_text(angle = 90,
-                                   vjust = 0.5,
-                                   hjust = 1,
-                                   size = 11)) +
-  scale_fill_discrete("") +
-  labs(title = sprintf("Aires de santé actives au %s",
-                       format(database_date, "%d %b %Y")),
-       x = "",
-       y = "Nombre de cas au cours des 7 derniers jours")
-```
-
-
-```{r current active table 7 days}
-
-current_active_7_days %>%
-  show_table()
-
-```
-
-
-```
-
-
-# Export data 
-
-We export some of the relevant tables, which will be placed in the current working directory:
-
-```{r export_xlsx}
-
-if (!dir.exists("produced_xlsx")) {
-  dir.create("produced_xlsx")
-}
-
-rio::export(inc_df,
-            file.path("produced_xlsx",
-                      "incidence_data_frame.xlsx"))
-
-rio::export(nb_as_active,
-            file.path("produced_xlsx",
-                      "active_health_areas_21_days.xlsx"))
-
-rio::export(current_active,
-            file.path("produced_xlsx",
-                      "current_active_health_areas.xlsx"))
-
-```
-
-
-
-
-# System information {.tabset .tabset-fade .tabset-pills}
-
-The following information documents the system on which the document was
-compiled.
-
-## System 
-
-This provides information on the operating system.
-
-```{r system_info}
-Sys.info()
-```
-
-## R environment
-
-This provides information on the version of R used:
-
-```{r R_session}
-R.version
-```
-
-
-## R packages
-
-This provides information on the packages used:
-
-```{r R_pkg}
-sessionInfo()
-```
+---
+title: "Incidence by aire de santé"
+author: "Madeleine Crowe, Flavio Finger, Thibaut Jombart, and Jonathan Polonsky for the epi analytic cell North Kivu"
+date: "`r format(Sys.time(), '%A %d %B %Y')`"
+output:
+  html_document:
+    toc: TRUE
+    toc_depth: 4
+    toc_float: true
+    toc_collapse: false
+    number_sections: true
+    highlight: pygments
+    theme: spacelab
+    code_folding: hide
+    css: !expr here::here('css', 'style.css')
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      collapse = TRUE,
+                      fig.width = 8,
+                      fig.height = 6,
+                      dpi = 150,
+                      warning = FALSE,
+                      message = FALSE)
+```
+
+
```

Comment:
Could you add the preamble from other reports e.g.
https://github.com/reconhub/report_factories_templates/blob/master/linelist_investigations/report_sources/epicurves_2019-10-12.Rmd#L20-L37

And add yourself as first author and maintainer (if you're okay with it)? You should be first author in any case.